### PR TITLE
DATAUP-753 change error text for select

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

Make a more generic bit of error text for missing select values.

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
- [x] Any dependent changes have been merged and published in downstream modules

# Updating Version and Release Notes (if applicable)

- [n/a] [Version has been bumped](https://semver.org/) for each release
- [x] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ define([
                     }
                 );
                 if (validation.diagnosis === Constants.DIAGNOSIS.REQUIRED_MISSING) {
-                    validation.errorMessage = 'Please select a value from the dropdown.';
+                    validation.errorMessage = 'A value is required.';
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js:84]: are there tests for this widget?

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js:84]: Yes, but they didn't test the exact error text in the validation message. I can add that.

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js:84]: Done!

**Review by ialarmedalien — APPROVED** [review]
Merge away!

**Pull request merged**